src/components/Modal/Modal.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Modal.css';
+
+const Modal = ({ title, message, onCancel, onConfirm }) => {
```

Comment:
Incapsulate all modal logic inside of component (state, functions to change state etc)